src/AzureStack/Admin/ComputeAdmin/Compute.Admin.Tests/src/Helpers/AzureStackTestBase.cs

```diff
@@ -85,7 +85,10 @@ public abstract class AzureStackTestBase<T> where T : class
 
         public void ValidateExpectedReturnCode(Action action, HttpStatusCode httpResponseCode)
         {
-            try { action.Invoke(); }
+            try {
```

Comment:
nit: Highly recommend using `Assert.Exception` here. Take a look at [this](https://stackoverflow.com/questions/45017295/assert-an-exception-using-xunit)

Reply:
Sure. Updated. Thanks.